src/detect-engine-analyzer.c

```diff
@@ -978,6 +978,32 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
     }
     jb_close(ctx.js);
 
+    jb_open_object(ctx.js, "policy");
```

Comment:
`policy` is good.
Do you want to give it a namespace as there are exception policies as well? e.g. `scope_policy` or something else..

Reply:
following up on this comment: maybe "match_policy"?